dist/featureFlags.js

```diff
@@ -216,4 +227,36 @@ angular.module('feature-flags').provider('featureFlags', function() {
     }];
 });
 
+angular.module('release-toggle', [])
```

Comment:
Yep, good idea, I like featur-flag-ui (its not really a dialog its just a div on the page)